internal/services/keyvault/key_vault_access_policy_data_source_test.go

```diff
@@ -0,0 +1,129 @@
+package keyvault_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/acceptance"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/acceptance/check"
+)
+
+type KeyVaultAccessPolicyDataSource struct{}
+
+func TestAccDataSourceKeyVaultAccessPolicy_key(t *testing.T) {
```

Comment:
same for tests
```suggestion
func TestAccKeyVaultAccessPolicyDataSource_key(t *testing.T) {
```

Reply:
https://github.com/hashicorp/terraform-provider-azurestack/pull/151/commits/380ae1ae98407c69f3c3a0ad66a69464ba6b0b2b fixed also